Taskfile.yml

```diff
@@ -82,26 +117,24 @@ tasks:
         {{ end -}}
 
   install-pre-commit:
-    # Someone might have this installed with `brew`, so only install if it can't be found.
+    # Someone might have this installed with `brew`,
+    # so only install if it can't be found.
     status:
       - which pre-commit
     cmds:
       - python3 -m pip install -U pre-commit
 
-  install-brew-dependencies:
-    status:
-      - which hugo
-      - which npm
-    preconditions:
-      - sh: which brew
-        msg: |
-
+  install-hugo:
+    cmds:
+      # - /home/linuxbrew/.linuxbrew/bin/brew install hugo
+      - curl -L https://github.com/gohugoio/hugo/releases/download/v0.91.2/hugo_0.91.2_Linux-64bit.deb -o hugo.deb
```

Comment:
One advantage of using `brew` is that it'll work on both Mac & Linux. 

My guess is that a majority of people are on MacOS? At the very least a decent proportion. So I don't think we should only provide Linux scripts. (Maybe stereotyping but Mac folks are also likely to benefit more from the scripts!)

Is there a reason we can't just `brew install hugo npm`?